### PR TITLE
Feat: Create User View for IDIR - 160

### PR DESCRIPTION
**Pull request by prv-proton**

### CODE REVIEW

**Comment by prv-proton** [comment]
- Footer styling
- Route mapping
- adjustments to the styling
- New User page redirect on clicking "New User" button in Admin > Users page.

File: frontend/src/layouts/authentication/components/Logout.jsx [hunk 1]
```diff
@@ -16,10 +16,15 @@ const Logout = (props) => {
...
+        justifyContent="space-around"
+        mr={2}
+      >
         <BCTypography
           variant="subtitle1"
-          color={props.isScrolled ? 'primary' : 'light'}
+          color={isScrolled ? 'primary' : 'light'}
           mx={1}
         >
           {user?.display_name}
```
Comment (kevin-hashimoto) [inline frontend/src/layouts/authentication/components/Logout.jsx:30]: error in console when this is empty. `<BCTypography/>` expects `children`.

```
{
  user?.display_name && (
    <BCTypography
          variant="subtitle1"
          color={isScrolled ? 'primary' : 'light'}
          mx={1}
        >
          {user?.display_name ?? ''}
        </BCTypography>
  )
}
```
will do

File: frontend/src/layouts/Layout.jsx [hunk 2]
```diff
@@ -19,65 +24,68 @@ const Layout = ({ crumbs }) => {
...
-                  )}
-                </Breadcrumbs>
-              </Paper>
-            )}
-            <BCBox py={4}>
-              <Outlet />
-              <Footer />
-            </BCBox>
-          </BCBox>
+        <Grid item lg={12}>
```
Comment (kevin-hashimoto) [inline frontend/src/layouts/Layout.jsx:75]: the item prop is throwing a console warning. we can probably remove it

File: frontend/src/routes.jsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                Welcome to the Compliance Report!
+              </BCTypography>
+            )
+          }
+        ]
+      },
+      // Organization pages for IDIR User
+      {
+        path: appRoutes.organization.main,
+        handle: { crumb: () => 'Organization' },
```
Comment (kevin-hashimoto) [inline frontend/src/routes.jsx:111]: we can remove this line as we dont need breadcrumbs on that page according to the mockup

**Review by kevin-hashimoto — CHANGES_REQUESTED** [review]
super minor changes that get rid of some console errors. otherwise ui looks good!

File: frontend/src/layouts/Layout.jsx (hunk 2 above)
Comment (prv-proton) [reply frontend/src/layouts/Layout.jsx:75]: yes, removed.

File: frontend/src/layouts/authentication/components/Logout.jsx (hunk 1 above)
Comment (prv-proton) [reply frontend/src/layouts/authentication/components/Logout.jsx:30]: thanks, fixed it.

File: frontend/src/routes.jsx (hunk 3 above)
Comment (prv-proton) [reply frontend/src/routes.jsx:111]: Need more info on this, we can discuss this more and change accordingly.

**Review by kevin-hashimoto — APPROVED** [review]
LGTM!

**Review by AlexZorkin — APPROVED** [review]

**Pull request merged**